plugins/power/meson.build

```diff
@@ -41,14 +41,16 @@ gsd_power = executable(
   install_dir: gsd_libexecdir
 )
 
-led_breathe = executable(
-  'led-breathe',
-  files('led-breathe.c'),
-  install: true,
-  install_mode: ['rwsr-xr-x', 'root'],
-  install_rpath: gsd_pkglibdir,
-  install_dir: gsd_libexecdir,
-)
+if meson.get_compiler('c').has_header('sys/io.h')
```

Comment:
At the toplevel `meson.build` file, there already is `cc = meson.get_compiler('c')`. That can be used here as `cc.has_header('sys/io.h')`